### PR TITLE
ICO calendar lisiting

### PR DESCRIPTION
**Pull request by richardmcb**
Hi there, 

I wanted to share our clients latest ICO with you to get on your ICO calendar. Details are;

Project name: Science Blockchain
Token name and ticker symbol: SCI
One line catch phrase: Blockchain startup incubator 
Concept description: 
Since Science started in 2011, they have co-founded and invested in more than 70 companies and exited multiple companies including Dollar Shave Club (acquired by Unilever for $1B), HelloSociety (acquired by New York Times) and FameBit (acquired by Google). They have a track record for identifying legacy industries and creating platforms for innovation. They are the first Incubator to launch an ICO with an incredible team behind them.
Science wants to use their expertise at company building to help select blockchain entrepreneurs in their efforts to become the software giants of the next century. They believe token holders should benefit from access to a stream of new tokens from Science’s carefully incubated portfolio companies. In turn, investors should as a result, participate in any upside from ownership of the Science Tokens, as the proceeds from dispositions of portfolio companies are anticipated to be used to repurchase those tokens via a buyback program.

Country of origin: USA
Website link: www.science.tokenhub.com
Token sale start date: Pre Sale 11 Sept 2017, Public Sale 2 Oct 2017
Token sale end date: 30 Oct 2017

### CODE REVIEW

**Comment by avadhootkulkarni** [comment]
The ICO Calendar is being maintained by me at https://github.com/avadhootkulkarni/UltimateICOCalendar

Please send a pull request there.

Thanks!

**Comment by nikshilov** [comment]
Hi!
I want to announce Galactikka PRE-ICO.

**Date**
September 04

**Asset**
GALA

**One Liner**
Social network with P2P ad platform and blockchain-based reputation system.

**Blockchain**
Ethereum

**Website**
[ICO website](http://ico.galactikka.com)

**Slack**
[Galactikka slack](https://galactikka-invite-ico.herokuapp.com/)

**Comment by hsatija1** [comment]
Hi there,

I wanted to share our clients latest ICO with you to get on your ICO calendar. Details are;

Project name: VuePay
Token name and ticker symbol: VUP
One line catchphrase: Democracy to Ad Ecosystem
Concept description: At VuePay our mission is to become the most favored platform for advertising directly to target audience where the users will get paid to watch ads and to tap into the billion plus dollar advertisement market using their smart devices. VuePay will be available on connected mobile devices regardless of geography, language or currency.
VuePay will cut the middlemen and target the viewers directly, which in turn will give advertisers better control in targeting markets and an auditable trail of transactions using the Ethereum Block chain technology.
By using Ethereum decentralized platform our aim is to deliver a solution that offers security, auditability, liquidity, traceability and ease of use.

Country of origin: UK
Website link: www.vuepay.co
Pre Sale: 6th October 2017 to 20th October 2017
ICO date: 21st October 2017 to 20th Novemeber 2017
Whitepaper: http://vuepay.co/wp-content/uploads/2017/06/VuePay%20whitepaper.pdf

Telegram: https://t.me/joinchat/E0PjYA2Rlgx_Mr2FzY42KQ

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ If you have any random suggestions (like adding a column), feel free to post an
...
 | September 28 | Neuron | Doc.ai's Blockchain-Enabled Natural Language Processing Platform For Quantified Biology | Ethereum | [Link](https://tokensale.doc.ai) | | [Link](https://www.reddit.com/r/doc_ai/) | [Link](https://twitter.com/_doc_ai) |
 | October 2 | Decision Token | Democratic tools for the 21st century | Ethereum | [Link](https://horizonstate.com/) |  | [Link](https://www.reddit.com/r/horizonstate/) | [Link](https://twitter.com/HorizonState) |
+| October 2 | RefToken | Making the blockchain mainstream, one block at a time | Ethereum | [Link](https://reftoken.io/) |  | [Link](https://www.reddit.com/r/RefToken/) | [Link](https://twitter.com/reftoken) |
 | November 8 | FintruX | Enables borrowers to create their own fully customized and simplified smart contracts with no code for every contract | Ethereum | [Link](https://www.fintrux.com) | [Link](https://gitter.im/FintruX/ICO-Discussion) | [Link](https://www.reddit.com/r/FintruX/) | [Link](https://twitter.com/FintruX) |
 
 ## Coming Soon
```
Comment (hsatija1) [inline README.md]: | October 6 | VuePay | Democracy to Ad Ecosystem | Ethereum | [Link](http://vuepay.co/) |  | [Link](https://www.reddit.com/r/vuepay) | [Link](https://twitter.com/vuepay) | [Link](https://facebook.com/vuepay) | [Link] https://t.me/joinchat/E0PjYA2Rlgx_Mr2FzY42KQ | [Link] Whitepaper: http://vuepay.co/wp-content/uploads/2017/06/VuePay%20whitepaper.pdf

**Comment by hsatija1** [comment]
Hi there,

I wanted to share our clients latest ICO with you to get on your ICO calendar. Details are;

Project name: VuePay
Token name and ticker symbol: VUP
One line catchphrase: Democracy to Ad Ecosystem
Concept description: At VuePay our mission is to become the most favored platform for advertising directly to target audience where the users will get paid to watch ads and to tap into the billion plus dollar advertisement market using their smart devices. VuePay will be available on connected mobile devices regardless of geography, language or currency.
VuePay will cut the middlemen and target the viewers directly, which in turn will give advertisers better control in targeting markets and an auditable trail of transactions using the Ethereum Block chain technology.
By using Ethereum decentralized platform our aim is to deliver a solution that offers security, auditability, liquidity, traceability and ease of use.

Country of origin: UK
Website link: www.vuepay.co
Pre Sale: 6th October 2017 to 20th October 2017
ICO date: 21st October 2017 to 20th Novemeber 2017
Whitepaper: http://vuepay.co/wp-content/uploads/2017/06/VuePay%20whitepaper.pdf

**Pull request closed without merging**